JsonLogic/Rule.cs

```diff
@@ -111,13 +121,13 @@ public override Rule Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSer
 					throw new JsonException($"Cannot identify rule for {op}");
 
 				rule = args is null
-					? (Rule)JsonSerializer.Deserialize("[]", ruleType, options)!
-					: (Rule)args.Deserialize(ruleType, options)!;
+					? (Rule)JsonSerializer.Deserialize("[]", ruleType, LogicSerializerContext.Default)!
```

Comment:
This is technically a behavior change because we're not passing the options along.

I would recommend passing options and suppressing the warning, as unsavory as that sounds.

Reply:
I actually just probably need to use the options extension.  Likely just missed this.